ding/envs/env_manager/gym_vector_env_manager.py

```diff
@@ -18,7 +18,7 @@ class GymVectorEnvManager(BaseEnvManager):
         Create an GymVectorEnvManager to manage multiple environments.
         Each Environment is run by a respective subprocess.
     Interfaces:
-        seed, ready_obs, step, reset
+        seed, ready_obs, step, reset, close
```

Comment:
set `self._closed = False` when create `self.env_manager`

Reply:
is it ok to set self._closed = False at the beginning of reset method?

Reply:
No, you can refer to the source code of gym, relevant components like subprocess are already created when calling construction method.